vscode/package.json

```diff
@@ -174,6 +174,13 @@
         "shortTitle": "Preview doc",
         "icon": "$(preview)"
       },
+      {
+        "command": "stencila.open-chat-assistant",
+        "category": "Stencila",
+        "title": "Open chat assistant",
+        "shortTitle": "Chat assistant",
```

Comment:
We are avoiding anthropomorphism and trying to take a more sober, matter-of-fact, approach to LLM use so avoiding "assistant" and "agent" at present:

```suggestion
        "command": "stencila.open-model-chat",
        "category": "Stencila",
        "title": "Open model chat",
        "shortTitle": "Model chat",
```